tools/dockerfile/grpc_artifact_python_manylinux2014_x64/Dockerfile

```diff
@@ -13,18 +13,21 @@
 # limitations under the License.
 
 # Docker file for building gRPC manylinux Python artifacts.
-# Updated: 2021-08-23
+# Updated: 2021-08-27
 
-# Use a tagged version to use Python 3.5
-FROM quay.io/pypa/manylinux2014_x86_64:2021-05-01-28d233a
+# Use a tagged version to use Python 3.10
```

Comment:
nit: what exactly does this sentence mean "Use a tagged version to use Python 3.10"?   to use Python3.10 where? how about the other python versions?
Is this supposed to mean "use a tagged version so that all of python 3.6-3.10 versions are included?"